contentscript.js

```diff
@@ -2,11 +2,20 @@ var tags = document.getElementsByClassName('tag');
 blocklist = ["jura"]
 
 counter = 0;
-for (var i=0; i < tags.length; i++) {
-    if(blocklist.includes(tags[i].textContent.toLowerCase())) {
-        console.log(`Matched tag: ${tags[i].textContent}`);
-        tags[i].closest(".questionbox").remove()
-        counter ++;
-    }
+// Sicherstellen, dass wir in den EF Thread übersichten sind
+// Deckt alles von Neuste bis Beliebteste Fragen ab
+// Meine Beiträge haben eine andere URL und das ist gut so.
+taburl = document.location.href;
+//console.log(`Scanning the following url for named tags ${taburl}`);
+
+if (taburl.includes("community.e-fellows.net/qanda/index/list")){
```

Comment:
Es gibt auch die URL https://www.community.e-fellows.net/qanda falls man nur auf "neueste Fragen" klickt, was essentiell das gleich wie diese URL sein sollte. 
```suggestion
if (taburl.includes("community.e-fellows.net/qanda")){
```

Reply:
Das war mir nicht bewusst, Du hast aber natürlich völlig Recht. 
Dein Change funktioniert aber sicher nicht. quanda ist im Prinzip der Name ihres CMS und das steckt in absolut jeder URL drin. Damit schränkst Du es nur auf Ihre Seite ein, was ohne Frage wertvoll ist, aber damit gehen wieder die Fragen selbst kaputt. Wir testen einfach beides.

Reply:
Habe eine Änderung hinzugefügt die das ursprüngliche Problem lösen sollte. Diese suggestion würde ich dann einfach so übernehmen, das ist nicht verkehrt denke ich.

Reply:
Beat me to it. Ich war gerade am Regex basteln, aber deine Lösung ist eh besser.